### DISCUSSION TITLE
2024-07-23 화요일

### DISCUSSION THREAD

**Discussion by Captain-Kim** [nbc-final-a10/TripBuddy#81] [기록보드]
2024-07-23 화요일 기록보드입니다.

## 여행가고십조(A10) 일일 공통 일과표

| 시간 | 일과 |  기록 |
| :-: | :-: | :-: |
| 09:00 ~ 10:00 | 일일 1차 정기 회의 | <img src="https://github.com/user-attachments/assets/82f1940b-1409-420f-82e1-fd41f7308d4a" height="150" witdh="150"/> |
| 10:00 ~ 10:30 | 하루 일과 준비 | |
| 10:30 ~ 13:00 | 개발 | |
| 13:00 ~ 14:00 | 점심 식사 | |
| 14:00 ~ 15:00 | 일일 2차 정기 회의 | |
| 15:00 ~ 18:00 | 개발 | |
| 18:00 ~ 19:00 | 저녁 식사 | |
| 19:00 ~ 20:00 | 일일 3차 정기 회의(merge time) | <img src="https://github.com/user-attachments/assets/4c4dc7ce-9217-4fab-a881-a8958aacd9b0" height="150" width="150"/> |
| 20:00 ~ 21:00 | 5분 기록보드 작성 / 개인 회고 | |

**Comment by Captain-Kim** [comment]
## 개인 기록보드 서식 (김병준)

### 오늘 한 일

- Funnel Pattern 버튼 커스텀 훅 작성


- 여정 작성 페이지 첫번째 페이지 UI 제작 (퍼널)

<img src="https://github.com/user-attachments/assets/a806d797-7c6e-41c3-bb71-eb23e8a6c4cc" width="300" height="300"/>

- Progress Indicator 작성

<img src="https://github.com/user-attachments/assets/937de875-91a4-4199-a17f-9d44175dc8a8" width="300" height="300"/>

### 트러블 슈팅

- 퍼널 커스텀 훅을 작성하면서 커스텀 훅에서 버튼의 스타일링까지 마치려고 했으나, TailwindCSS에서 커스텀 훅 내부의 className을 인식하지 못함 -> 커스텀 훅을 사용하는 페이지에서 동적으로 스타일링하여 prop을 내리는 방식으로 해결

```tsx
<NextButton className="text-2xl bg-main-color font-bold py-2 px-4 rounded" />
```

- 위 사항에 대한 이강민 튜터님 피드백 : clsx 라이브러리를 결합하여 사용

### 내일 계획

**Comment by eunohhh** [comment]
### 오늘 한 일
- 인증/인가 기능구현 - 로그인, 로그아웃, sns로그인, 회원가입 까지 완료
- 장소, 테마, mbti 데이터 및 타입 세팅
- 온보딩 페이지 기능부터 구현 시작

### 내일 계획
- cva 칩스 작성
- 인증/인가 완성(온보딩은 빼고)
- 스토리 페이지 작성 시작

**Comment by slsl2** [comment]
# 오늘 한 일

- 메인 페이지 와이어프레임 UI
- 탭메뉴 와이어프레임 UI
- 헤더 와이어프레임 UI
- 컴포넌트 분리 및 재사용을 최대한 고려하며 작성
- 터미널 -4094 문제 해결 : 자동 새로고침을 off로 설정해서 해결

**Comment by hyunjeongkwak** [comment]
### 오늘 한 일
- 검색 1페이지 375px UI 제작(1280px은 와이어프레임 확정 후 제작 예정)
- 검색 2페이지 375px UI 제작(1280px은 와이어프레임 확정 후 제작 예정)
- 여행테마와 버디즈 속성 정보 렌더링, 로케이션 정보 렌더링

### 내일 할 일
- 인풋창 정리
- 퍼널 패턴 적용해 페이지 제작
- 캘린더 라이브러리 사용해 날짜 선택 페이지 제작
- 공용 컴포넌트 공유 및 적용

**Comment by Kyara0324** [comment]
### 오늘 한 일
- 튜토리얼 페이지 ui 제작
### 내일 할 일
- Tutorial 페이지 퍼널패턴 적용 및 폴더구조 정리
